### PR TITLE
Bug 1220540 - Revise disconnect page style.

### PR DESCRIPTION
**Pull request by danhuang1202**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e926a87a06b043401a60bc14f0f8f1f1ae65e3d8

File: tv_apps/browser/js/browser_dialog.js [hunk 1]
```diff
@@ -256,17 +257,20 @@ var BrowserDialog = {
       this.focusIndex.y = 0;
       this.focusElement[0][countIndex++] = this.browserDialogButton1;
     }
+
+//IFDEF_FIREFOX_SYNC
+      if(type === 'signout_confirm') {
+        this.focusIndex.y = 0;
+      }
```
Comment (begeeben) [inline tv_apps/browser/js/browser_dialog.js]: Indentation for these 3 lines is incorrect.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1a47530d1dadd8c2f56fcd9ad9d294e12bef9838

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8484eb7cc7593e6790b5cf2bb1921400a260f199

**Pull request merged**